lib/ansible/modules/commands/command.py

```diff
@@ -53,7 +53,7 @@
   removes:
     type: path
     description:
-      - A filename or (since 2.0) glob pattern. If it already exists, this step B(will) be run.
+      - A filename or (since 2.0) glob pattern. If it still exists, this step B(will) be run.
```

Comment:
```suggestion
      - A filename or (since 2.0) glob pattern. If it exists, this step B(will) be run.
```

Reply:
Thanks, I took your suggestion and enhanced it because globs match files, they don't exist.